src/pulse3D/plate_recording.py

```diff
@@ -557,10 +557,14 @@ def to_dataframe(self) -> pd.DataFrame:
 
         data = {"Time (s)": pd.Series(interp_timepoints)}
 
-        output_stim_data = not self.is_optical_recording and first_well.version >= VersionInfo.parse("1.0.0")
+        output_stim_data = (
+            not self.is_optical_recording
+            and first_well.version >= VersionInfo.parse(MIN_FILE_VERSION_FOR_STIM_INTERPOLATION)
+            or include_stim_data
```

Comment:
`and include_stim_data`. Using `or` here wouldn't make sense since it could try to output stim data from a optical file or beta 1 file